<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>CreateComponent</class>
    <widget class="QWidget" name="CreateComponent">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>640</width>
                <height>480</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Create Component</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QLabel" name="label">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string>Type:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QComboBox" name="comboTypes"/>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                        <widget class="QLabel" name="label_2">
                            <property name="text">
                                <string>Label:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLineEdit" name="lineLabel"/>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QFrame" name="frameExtra">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="frameShape">
                        <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                </widget>
            </item>
            <item>
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>40</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <widget class="QPushButton" name="buttonCreate">
                    <property name="text">
                        <string>Create Component</string>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
